### PR TITLE
67 api create restfull routes for tokens using coingecko

### PR DESCRIPTION
**Pull request by MTthoas**
@Ronfflex @Guiiz94 Check that we have all the necessary roads for your uses

### CODE REVIEW

File: api/queries/token.query.go [hunk 1]
```diff
@@ -1,39 +1,100 @@
...
 	"os"
+
+	"github.com/MTthoas/dex/api/models"
 )
 
-func ScrapeEthereumTokens() []models.Token {
+func GetTokens() []models.Token {
 
-	apiUrl := os.Getenv("API_URL")
+	apiUrl := "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&category=ethereum-ecosystem"
```
Comment (Ronfflex) [inline api/queries/token.query.go:16]: Why removing it from the .env file ? Should stay there no ?

**Comment by Ronfflex** [comment]
good to go

File: api/queries/token.query.go (hunk 1 above)
Comment (MTthoas) [reply api/queries/token.query.go:16]: Its only the URL, credentials are above in the header `x-cg-demo-api-key` key. 
We've have ton only change `CG_API_KEY`key in the new .env

We can still change the `vs_currency`value in the URL or put in a .env value to change currency.
But its not necessary, we only manipulates USD values for our use cases

Comment (Guiiz94) [reply api/queries/token.query.go:16]: I think we can leave so, the url has no sensitive data

**Pull request merged**